app/routes/index.js

```diff
@@ -23,16 +22,29 @@ module.exports = {
 }
 
 const getNotifications = async (sbi) => {
```

Comment:
I think its time to split these functions out into their own files and require them in.

Reply:
Ah yeah, will get this done. Thanks.

Reply:
All done, have tested sending a couple messages from Service Bus and all working good.